### ISSUE TITLE
Notification handler 'textDocument/didOpen' failed with message: Cannot read property 'token' of undefined

### ISSUE THREAD

**Issue by chrissena** [angular/vscode-ng-language-service#52]
I have installed the latest version of vscode- ng-language-service (0.1.0) but for some reason it does not work at all. when I open any file nothing appears to happen. However when I look at the  output i see...

![image](https://cloud.githubusercontent.com/assets/6528986/24331083/fdf2a4fa-1224-11e7-835e-abbe405b415c.png)

I am using typescript 2.2.1 and angular-cli 1.0.0.
Any ideas as to what is causing this?
Thanks

**Comment by chuckjaz** [comment]
I am not sure. I don't use `token` in my server but I note that the `vscode` modues I use use this. I will change this diagnostic to include the stack in the next release to be able to debug this more.

Do you have a repository I can clone that reproduces this?

**Comment by chrissena** [comment]
I've just installed version 0.1.1 and still getting the error. I will see if I can try and create a repo later this week.

This is the new output with stack i'm getting in the latest version.
![image](https://cloud.githubusercontent.com/assets/6528986/24643982/fd49949c-1908-11e7-8e98-1b469ddaca80.png)

**Comment by chuckjaz** [comment]
I believe I see the issue. Can you put a `return` statement just after the call to `.reportError(syntaxError("Invalid " + ...` in `language-service.umd.js`?

**Comment by chuckjaz** [comment]
https://github.com/angular/angular/issues/15759

**Comment by martinmcwhorter** [comment]
I am seeing a similar error:

```
Search path: c:/dev/some-app/src/app/...
Config file name: c:/dev/some-app/tsconfig.json
Failed on type {"filePath":"c:/dev/some-app/src/app/layout/menu/menu.component.ts","name":"MenuComponent","members":[]} with error TypeError: Cannot read property 'forEach' of undefined
[Error - 5:17:24 PM] Notification handler 'textDocument/didOpen' failed with message: Cannot read property 'forEach' of undefined
```
Let me know if this is unrelated and I can open a new issue.

**Comment by chuckjaz** [comment]
@martinmcwhorter This appears to be a different issue. Can you open a new issue?

**Comment by chuckjaz** [comment]
Fixed in `0.1.2`
